content/en/guide/v10/getting-started.md

```diff
@@ -63,7 +63,7 @@ Writing raw `h` or `createElement` calls can be tedious. JSX has the advantage o
 >
 > `import { html, render } from 'https://esm.sh/htm/preact/standalone'`
 
-For a more full example, see [Using Preact with HTM and ImportMaps](#using-preact-with-htm-and-importmaps), and for more information on HTM, check out its [documentation][htm].
+For a more scalable solution, see [Import Maps -- Basic Usage](/guide/v10/no-build-workflows#basic-usage), and for more information on HTM, check out its [documentation][htm].
```

Comment:
Do we want to refer to import maps as more scalable? Obviously HTTP imports have "issues", but I'm not sure of the language here.